.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: Tackle Pathfinder CI main
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  unit-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.5
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        java-package: jdk
+    - name: Cache Maven packages
+      uses: actions/cache@v3.0.1
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Build with Maven
+      run: ./mvnw -U -B test
+
+  integration-native-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2.3.5
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        java-package: jdk
+    - name: Cache Maven packages
+      uses: actions/cache@v3.0.1
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Build with Maven
+      run: ./mvnw -U -B verify -Pnative -Dquarkus-profile=test -Dquarkus.native.additional-build-args=--allow-incomplete-classpath
+
+  code-coverage:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2.3.5
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          java-package: jdk
+      - name: Cache Maven packages
+        uses: actions/cache@v3.0.1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Generate jacoco report
+        run: ./mvnw -U verify -Pjacoco
+      - uses: codecov/codecov-action@v1.5.0
+        with:
+          file: ./target/site/jacoco/jacoco.xml
+          name: codecov
```

Comment:
overachiever?

Reply:
No, this isn't me. :) The folks maintaining Pathfinder already had this. I just moved push.yml to ci.yml. For whatever reason github is showing it as a removal of one and creation of the other. Perhaps because of the large chunk for the image build/push removed.